src/utils/ShareButton/shareTracking.js

```diff
@@ -0,0 +1,10 @@
+const shareTracking = channel => {
+  const dataLayer = window.dataLayer || [];
```

Comment:
This needs to be a reference to the global `dataLayer` object

Something like this would work

```
if (!window.dataLayer) {
  window.dataLayour = [];
}

window.dataLayer.push({
    event: 'btn_social-share',
    action: channel
});
```

At present if `window.dataLayer` isn't already defined, we'd just be adding the event to a new array which will be chucked away at the end of the function.